Library/Homebrew/caveats.rb

```diff
@@ -46,6 +47,14 @@ def zsh_completion_caveats
     end
   end
 
+  def fish_completion_caveats
+    if keg and keg.completion_installed? :fish and which("fish") then <<-EOS.undent
```

Comment:
Let's use `&&` instead of `and` here, thanks and consider moving the `<<-EOS.undent` onto a new line.